tests/rptest/tests/partition_balancer_test.py

```diff
@@ -75,6 +75,61 @@ def node2partition_count(self):
 
         return ret
 
+    def get_partition_amount_on_node(self, node):
+        admin = Admin(self.redpanda)
+        return len(
+            list(
+                filter(lambda x: x["ns"] == "kafka",
+                       admin.get_partitions(node=node))))
+
+    def validate_metrics_node_down(self, disabled_node,
+                                   expected_moving_partitions_amount):
+        self.logger.debug(
+            f"Checking metrics for movement with disabled node: {self.redpanda.idx(disabled_node)}"
+        )
+        availabale_nodes = list(
+            filter(lambda x: x != disabled_node, self.redpanda.nodes))
+        metrics = self.redpanda.metrics_sample(
```

Comment:
Nit: You could use `redpanda.metrics_samples` to get the values from multiple metrics in a single call. Currently, each call to `redpanda.metrics_sample` sends a new query to the metrics endpoint. It's a minor point though. Feel free to ignore.